scripts/generate_landings_risk_report.py

```diff
@@ -621,10 +622,10 @@ def get_commit_data(commit_list: list[repository.CommitDict]) -> list[dict]:
                             and change["removed"] == "UNCONFIRMED"
                             and change["added"] in ("NEW", "ASSIGNED")
                         ):
-                            time_to_confirm = (
-                                dateutil.parser.parse(history["when"])
-                                - dateutil.parser.parse(bug["creation_time"])
-                            ).total_seconds() / 86400
+                            time_to_confirm = business_day_range(
+                                dateutil.parser.parse(history["when"]),
+                                dateutil.parser.parse(bug["creation_time"]),
```

Comment:
Is it necessary to parse the date here?

Reply:
I'm not quite sure why I did it this way to be honest, I had a reason but it might have been for before I switched to the numpy implementation.